### PR TITLE
Generating API with STA

### PR DESCRIPTION
**Pull request by jacob-b-labs**

### CODE REVIEW

File: __tests__/apiClient.test.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { Api, HttpResponse } from "../api/Api";
+
+describe('apiClient', () => {
+  it('downloads a list of events', async () => {
+    const fetch = require('node-fetch');
```
Comment (Mattioo7) [inline __tests__/apiClient.test.ts:5]: what does it do?

File: __tests__/apiClient.test.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    const client = new Api({
+      baseUrl: "http://127.0.0.1:4010",
+      customFetch: fetch as any
```
Comment (Mattioo7) [inline __tests__/apiClient.test.ts:9]: Same as above

**Review by Mattioo7 — APPROVED** [review]

File: __tests__/apiClient.test.ts (hunk 1 above)
Comment (jacob-b-labs) [reply __tests__/apiClient.test.ts:5]: Dynamic import for fetch, used because Jest doesn't have its own.

File: __tests__/apiClient.test.ts (hunk 2 above)
Comment (jacob-b-labs) [reply __tests__/apiClient.test.ts:9]: Same reason as above. Normal development doesn't require customFetch at all.

**Pull request merged**